<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.909, stop:0 rgba(193, 130, 255, 255), stop:0.97619 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;URW Chancery L&quot;;
color: rgb(108, 0, 81);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Catalogue">
     <attribute name="title">
      <string>Catalogue</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelCatalogue">
           <property name="text">
            <string>Catalogue</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableViewCatalogue">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAjoutProduit">
           <property name="text">
            <string>Nouveau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSupprProduit">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Commandes">
     <attribute name="title">
      <string>Commandes</string>
     </attribute>
     <widget class="QLabel" name="label_Client">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>59</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewCommande">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>561</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNouveauClient">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxProduit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>541</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Produit</string>
      </property>
      <widget class="QTableWidget" name="tableWidgetProduit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>451</width>
         <height>171</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Num</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Libellé</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prix</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Quantité</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButtonValider">
       <property name="geometry">
        <rect>
         <x>461</x>
         <y>50</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>451</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="comboBoxProduit">
          <item>
           <property name="text">
            <string>pneu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>crash cage</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelQuantite">
          <property name="text">
           <string>Quantité</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxQuantite"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAjouterProd">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>541</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Panier</string>
      </property>
      <widget class="QTableWidget" name="tableWidgetPanier">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>531</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonEnregistrerPanier">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>87</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;URW Chancery L&quot;;</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>152</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_Id">
           <property name="text">
            <string>Identifiant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Nom">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Prenom">
           <property name="text">
            <string>Prénom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_Prenom"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Nom"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditId"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_Rue">
           <property name="text">
            <string>Rue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Ville">
           <property name="text">
            <string>Ville</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCP">
           <property name="text">
            <string>Code Postal</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditCP"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Ville"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Rue"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Client">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QTableView" name="tableViewClient">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRechClient">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEvenement">
    <property name="title">
     <string>Evenement</string>
    </property>
    <addaction name="actionGestion_du_catalogue"/>
    <addaction name="actionEnregistrement_des_commandes"/>
    <addaction name="actionClient"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEvenement"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGestion_du_catalogue">
   <property name="text">
    <string>Gestion du catalogue</string>
   </property>
  </action>
  <action name="actionEnregistrement_des_commandes">
   <property name="text">
    <string>Enregistrement des commandes</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
